contents/blog/first-ops-hire.mdx

```diff
@@ -0,0 +1,72 @@
+---
+date: 2023-09-07
+title: Making your first startup ops hire – what founders should look for
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - charles-cook
+featuredImage: ../images/blog/posthog-blog-image.png
+featuredImageType: full
+category: Startups
+tags:
+  - Guides
+---
+
+I've previously written about [startup hiring strategy](/blog/early-stage-startup-hiring-strategy) more generally. This article covers a specific role – what to look out for when making your first ops hire into your startup, aka probably your first non-engineer. Usually this person is hire number 5-10, and they will take over stuff that one of the cofounders has been doing up to that point.
```

Comment:
intro is boring!

Reply:
Yeah I was trying to build credibility/explain why you should listen to me. How about:

'Most founders I talk to approach their first ops hire in a totally sensible but wrong way. Their typical approach is to look at the long list of admin tasks they have accumulated, cry a bit, and then look for someone who has done those things before, plus a bit extra. However, the reality is that attributes >>> technical expertise when it comes to your first ops hire.'

Alternative approach - just delete this section and cut straight to 'Your first ops hire will have a disproportionate impact on your company culture.'